verticapy/performance/vertica/collection/collection_tables.py

```diff
@@ -54,7 +57,61 @@ class BundleVersion(Enum):
     """
 
     V1 = 1
-    LATEST = V1
+    V2 = 2
+    LATEST = V2
+
+
+class TableMetadata:
+    """
+    ``TableMetadata`` holds information about a parquet file
+    that represents a table. It can be serialized to JSON
+    and read back later.
+    """
+
+    # TODO: someday this class should also be able to
+    # de-serialize data.
+    def __init__(self, file_name: Path, table_type: AllTableTypes, exported_rows: int):
+        self.file_name = file_name
+        self.table_type = table_type
+        self.exported_rows = exported_rows
+
+    def to_json(self):
+        return {
+            "table_type_name": str(self.table_type.name),
+            "table_type_value": str(self.table_type.value),
+            "file_name": str(self.file_name),
+            "exported_rows": self.exported_rows,
+        }
+
+
+class ExportMetadata:
+    """
+    ``ExportMetadata`` contains all of the metadata for a export
+    bundle of parquet files. It has methods to write the metadata
+    to a file.
+    """
+
+    # TODO: someday this class should also be able to de-serialize
+    # data from a json file.
+    def __init__(
+        self, file_name: Path, version: BundleVersion, tables: List[TableMetadata]
+    ):
+        self.file_name = file_name
+        self.version = version
+        self.tables = tables
+
+    def to_json(self):
+        return {
+            "version": str(self.version.value),
+            "tables": [x.to_json() for x in self.tables],
+        }
+
+    def write_to_file(self):
```

Comment:
We need precise description for each function. Many docstrings are missing in the PR.

Reply:
Added a docstring for this function and class